test/v1/integration_test.go

```diff
@@ -273,7 +273,7 @@ func mustMakeDisperser(t *testing.T, cst core.IndexedChainState, store disperser
 		dockertestPool, dockertestResource, err = deploy.StartDockertestWithLocalstackContainer(localStackPort)
 		if err != nil {
 			teardown()
-			panic("failed to start localstack container")
+			panic("failed to start localstack container" + err.Error())
```

Comment:
nit: panic string should have a trailing space, to be formatted correctly with the appended error

Reply:
fixed everywhere: [2d5c409](https://github.com/Layr-Labs/eigenda/pull/1765/commits/2d5c40959759812f714cf54ad93dd093fe49f66b)